mergify_engine/rules/__init__.py

```diff
@@ -289,18 +397,28 @@ async def create(
 
             await rule.conditions.eval(ctxt.pull_request)
 
-            for condition in rule.conditions:
-                if hide_rule and condition.evaluation_error:
-                    self.faulty_rules.append(typing.cast(T_EvaluatedRule, rule))
-                    break
-                elif (
-                    hide_rule
-                    and condition.get_attribute_name() in self.BASE_ATTRIBUTES
-                    and not condition.match
-                ):
+            categorized = False
```

Comment:
can you explain?

Reply:
In the summary, we display rules in four groups:
* rules where all attributes match
* rules where only `BASE_ATTRIBUTES` match (filter out rule written for other branches)
* rules that won't work due to a configuration issue detected at runtime (eg team doesn't exists)
* rules where only `BASE_ATTRIBUTES` don't match (mainly to hide rule written for other branches)

Reply:
Ok, can you add this as a comment in the code so it's easier for understand to everyone passing by? 😉